src/utils_latency.c

```diff
@@ -126,8 +126,8 @@ latency_counter_t *latency_counter_create (void) /* {{{ */
     return (NULL);
   memset (lc, 0, sizeof (*lc));
 
-  latency_counter_reset (lc);
   lc->bin_width = HISTOGRAM_DEFAULT_BIN_WIDTH;
+  latency_counter_reset (lc);
```

Comment:
Oh, good point. I think that should actually be fixed in counter_reset(). Otherwise, it's very easy to accidentally break this in the future with an unrelated change.